clipper_admin/clipper_admin/deployers/caffe2.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import print_function, with_statement, absolute_import
+import shutil
+import torch
+import logging
+import re
+import os
+import json
+
+from ..version import __version__
+from ..clipper_admin import ClipperException
+from .deployer_utils import save_python_function, serialize_object
+
+logger = logging.getLogger(__name__)
+
+
+def create_endpoint(
```

Comment:
Rename this to `create_pytorch_endpoint`, add an argument called `onnx_backend="caffe2"`, then change the `base_image` default value to `None` (`base_image=None`).

Some context: We might want to support multiple ONNX backends soon. The cool thing about ONNX is that it decouples the choice of backend from the source of the model. So within the onnx model deployer we can support multiple training frameworks and inference frameworks. E.g. along with our `create_pytorch_endpoint` we might have a `create_caffe2_endpoint` and a `create_mxnet_endpoint`, any of which could deploy their model to a Caffe2 backend or an mxnet backend or a pytorch backend.